app/models/cdx/client.rb

```diff
@@ -7,25 +7,27 @@ def initialize(opts={})
 
   def default_opts
     {
-      :wsdl => "https://devngn.epacdxnode.net/cdx-register/services/RegisterSignService?wsdl",
-      :pretty_print_xml => true,
-      :log => true,
-      :soap_version => 2,
-      :convert_request_keys_to => :none,
+      wsdl: "https://devngn.epacdxnode.net/cdx-register/services/RegisterSignService?wsdl",
```

Comment:
Should this URL maybe be an env var instead? Or some config option. I am imagining it would change per-environment.